std_msgs/Header header

bool gnss_fix_ok
bool diff_soln
bool rel_pos_valid
bool float_carr_soln
bool fixed_carr_soln
bool is_moving
bool ref_pos_miss
bool ref_obs_miss
bool rel_pos_heading_valid
bool rel_pos_normalized
uint16 flags